environment-dev.yml

```diff
@@ -7,19 +7,20 @@ dependencies:
   - ele
   - foyer
   - garnett>=0.7.1
-  - gsd>=1.2
```

Comment:
I my have missed the discussion around this, but what is the reason for updating the pins?

Reply:
No you didn't miss anything. Sorry, I should've mentioned this in the summary! I was having trouble building the dev environment locally and tried first removing the pins. I think pre-commit maybe uses a different libcxx than hoomd2.9.4 is compatible with? Anyway changing that pin fixed my issue. I can certainly change the others back.

Reply:
HOOMD 2.9.6 has some bugfixes with cxx libraries [changelog](https://github.com/glotzerlab/hoomd-blue/blob/master/CHANGELOG.rst#v296-2021-03-16) I'm no expert on this, but I know that 2.9.6 has fewer package conflicts than 2.9.4. 😅